Update com.github.macadmins.Nudge.plist

BigMacAdmin opened the pull request:
- Updated `pfm_macos_min` because the latest Nudge app has a macOS 12 requirement [source](https://github.com/macadmins/nudge/wiki/v2.0-features#breaking-changes)
- Changed `pfm_default` for `Random Delay` to `true` to match Nudge 2.0.2+ behavior [source](https://github.com/macadmins/nudge/releases/tag/v2.0.2.81720)
- Removed `pfm_value_placeholder` for `Random Delay` as it should no longer be needed.

relgit commented on Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist:
```diff
@@ -13,9 +13,9 @@
+	<date>2025-02-13T00:00:00Z</date>
 	<key>pfm_macos_min</key>
-	<string>11.0</string>
+	<string>12.0</string>
```
This should go back to 11.0 because the manifest includes keys for 1.x versions of Nudge which was not limited to 12.0 and above.

relgit reviewed (changes requested):
Thanks for the PR @BigMacAdmin. I made one comment on the changes, but also, Nudge 2.0 introduced `nudgeMajorUpgradeEventLaunch` and `nudgeMinorUpdateEventLaunchDelay`.

Could you add those too?

BigMacAdmin commented:
Reverted pfm_macos_min to v11

Reviewed Ari's comment rregarding `nudgeMajorUpgradeEventLaunch`. I did a github search on the Nudge repository, and determined that this is not a key, it's a mistake in documentation. `nudgeMajorUpgradeEventLaunchDelay` (notice "Delay") and `nudgeMinorUpdateEventLaunchDelay` are new keys.

These keys already exist in the Nudge manifext, however they were NOT included in `pfm_segments > UX` like other keys of this type. So I added them there.
